TextFieldEffects/TextFieldEffects/AkiraTextField.swift

```diff
@@ -8,6 +8,7 @@
 
 import UIKit
 
+@objc
```

Comment:
We don't need this change, if there are methods that need it. Please add it next to their declarations

```swift
@objc func something()
```